### ISSUE TITLE
Configuration UI Improvements

### ISSUE THREAD

**Issue by EduardMe** [NotePlan/plugins#215]
Here we collect UI improvements for the configuration screens:

- the cancel and save & close buttons should be next to each other
- the save button doesn't really need the "& close" part as the save action already closes
- hitting return within an edit field creates a new line, without resizing the field, thus making it appear blank
- pressing tab in an edit field should move to the next field, not insert a tab at the cursor position

**Comment by jgclark** [comment]
David suggests adding the ability to trigger the Folder picker where relevant, to make sure that only valid folder names are used. It also makes it easier for the user.

**Comment by dwertheimer** [comment]
...and file picker when you are specifying a specific file name/title

**Comment by EduardMe** [comment]
The folder/file picker is the command bar search, right?

**Comment by jgclark** [comment]
Yes, it could be, if it was constrained to the one task of files or folders.
